ci/acceptance-test/tests/cmd_console_test.go

```diff
@@ -0,0 +1,36 @@
+// +build acceptance
+
+// +build acceptance
+
+package tests
+
+import (
+	"strings"
+	"testing"
+	"os/exec"
+	"time"
+
+	"github.com/tidwall/gjson"
+)
+
+func TestCmdConsole_ShowOption(t *testing.T) {
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc")
+	instance_id := strings.TrimSpace(stdout.String())
+
+	waitUntil(t, 5 * time.Minute, func()error {
+		cmd := exec.Command("openvdc", "show", instance_id)
+		buf, err := cmd.CombinedOutput()
+		if err !=nil {
+			return err
+		}
+		result := gjson.GetBytes(buf, "instance.lastState.state")
+		if result.String() == "RUNNING" {
+			return nil
+		}
+		time.Sleep(5 * time.Second)
+		return WaitContinue
+	})
```

Comment:
Thank you for writing this. This will be very useful for future tests. I think it should be moved to a function in another file so we can re-use it but that can be done in another PR.